packages/editor-ui/src/components/SqlEditor/SqlEditor.vue

```diff
@@ -51,31 +92,76 @@ export default defineComponent({
 	data() {
 		return {
 			editor: {} as EditorView,
+			expressionsDocsUrl: EXPRESSIONS_DOCS_URL,
+			isFocused: false,
+			skipSegments: ['Statement', 'CompositeIdentifier', 'Parens'],
 		};
 	},
+	watch: {
```

Comment:
Can we combine the watcher and the computed property with something like `'ndvStore.ndvInputData'()`? (Not sure)

Reply:
Yeah, we can watch store state directly, without adding computed prop. Came to my mind but I forgot to change it after.